### DISCUSSION TITLE
Do use Map? Tell me!

### DISCUSSION THREAD

**Discussion by xclud** [xclud/flutter_map#41] [Show and tell]
If you use Map for something, please chime in here and tell me more!

I will be happy to see screenshots of your project to see the awesomeness you have done!

If you want your project listed in the README.md, tell me to do. And provide some links.

**Comment by nimr77** [comment]
Yeah, I use it for dashboards on my projects so the admin can edit the markers on the map and set their locations, size, and the icon of the markers.

**Comment by Leo-Chan01** [comment]
I just added it to my project, when I enjoy it, I will definitely come back here to thank you

**Comment by mahsakhaje** [comment]
using Map was easy and worked well on all platforms I used. you have done a great job.
hope to see you providing more useful packages :)

**Comment by devmuaz** [comment]
To be honest with you, I've been struggling with finding the best Map package that would work perfectly on Huawei devices without any issues (since huawei_map package has issues that could result in Google Play rejections).

But idea of this one is awesome, literally the best option I have for now.. and GREAT work you've done mate!
Thank You ❤️

**Comment by xclud** [comment]
@devmuaz You've melt my heart. Thank you!

**Comment by nilsmagnus** [comment]
I use it for a hobbyproject to display ship-positions from ais-data. Love it!

**Comment by wiwiaR** [comment]
Hi! I have a list of coordinates and want to create a vector to exhibit in a map. Can your package resolve my issue?

**Reply by xclud** [reply]
@wiwiaR Sure!

**Reply by wiwiaR** [reply]
Yeah, it worked! Thanks a lot. By the way, how do I set the map to start at a specific location? I want to open the map at my location, but I didn't find how to in the code.

**Reply by xclud** [reply]
When initializing the `MapController` you have to pass in a center coordinate.
